### PR TITLE
Adds errata links to 4.13 MS release notes

### PR DESCRIPTION
**Pull request by opayne1**
OSDOCS#5564:Adds errata links to 4.13 MS release notes 

includes other formatting and tweaks

Version(s):
4.13

Issue:
https://issues.redhat.com/browse/OSDOCS-5564

Link to docs preview:
https://59244--docspreview.netlify.app/microshift/latest/microshift_release_notes/microshift-4-13-release-notes.html#microshift-4-13-0-dp

QE review:
Probably still need to get a QE review with the addition of `sudo` to a command in the KI section.

Additional information:
Links will not work until the release date

### CODE REVIEW

File: microshift_release_notes/microshift-4-13-release-notes.adoc [hunk 1]
```diff
@@ -86,7 +86,10 @@ For more information, see xref:../microshift_support/microshift-etcd.html#micros
 [id="microshift-4-13-storage"]
 === Storage
 
-* {product-title} configures LVMS dynamically based on the Volume Groups present. For more details, read the xref:../microshift_storage/microshift-storage-plugin-overview.adoc#lvms-volume-group-name_microshift-storage-plugin-overview[LVMS system requirements] section of this documentation.
+[id="microshift-4-13-lvms-system-requirements"]
+==== LVMS system requirements
```
Comment (opayne1) [inline microshift_release_notes/microshift-4-13-release-notes.adoc:90]: Added a heading to this storage entry

**Comment by GroceryBoyJr** [comment]
Woops! 
/label peer-review-done
/remove-label peer-review-needed
@GroceryBoyJr : snatching defeat from the jaws of success....*sigh*

**Pull request merged**